### PR TITLE
fix: clarified OS version

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

Bugfix: clarifies OpenSearch version to avoid confusion.
Clarification: added note on OR1 instance usage

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by xevien96 — CHANGES_REQUESTED** [review]
I think we should be straight forward and say that OR1 instances are not supported at the moment. So we leave no wiggle room for the customer.

When running Amazon OpenSearch 2.11 or higher, we do not support [OR1 instances](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/or1.html) due to the [limitation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/or1.html#or1-considerations) for the index refresh interval. More information can be found [here](https://docs.camunda.io/docs/next/self-managed/setup/guides/using-existing-opensearch/)

**Comment by igpetrov** [comment]
> I think we should be straight forward and say that OR1 instances are not supported at the moment. So we leave no wiggle room for the customer.
> 

Okay, I see what you mean

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -88,16 +88,20 @@ Requirements for the components can be seen below:
...
+| Optimize    | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
 | Connectors  | OpenJDK 21+  |                                                                                                                                                                                                                   |
 | Web Modeler | -            | PostgreSQL 13.x, 14.x, 15.x, 16.x or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x                                                                                                                              |
 
 When running Elasticsearch, you must have the [appropriate Elasticsearch privileges](/self-managed/concepts/elasticsearch-privileges.md).
 
+When running Amazon OpenSearch 2.11 and higher on the [OR1 instances](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/or1.html),
+consider the indexes refresh interval limitation. Camunda components may fail to start if the index refresh interval
+is set below minimal instance value.
+
```
Comment (igpetrov) [inline docs/reference/supported-environments.md:104]:
```suggestion
When running Amazon OpenSearch 2.11 or higher, we do not support [OR1 instances](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/or1.html) due to the [limitation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/or1.html#or1-considerations) for the index refresh interval. More information can be found [here](https://docs.camunda.io/docs/next/self-managed/setup/guides/using-existing-opensearch/)
```

**Review by xevien96 — APPROVED** [review]
Looks good to me

**Comment by igpetrov** [comment]
Hello @camunda/tech-writers 

This one is ready to be merged after review.

**Review by conceptualshark — APPROVED** [review]
🚀

**Pull request merged**